### PR TITLE
Update HBL link

### PR DESCRIPTION
**Pull request by irowiki**
Release got changed to a zip instead of a 3dsx. Add blurb about what to download and where to put it.

### CODE REVIEW

File: cogs/assistance-cmds/boot3dsx.3ds.md [hunk 1]
```diff
@@ -3,4 +3,8 @@ help-desc: Download link for 3DS Homebrew Launcher, boot.3dsx
 aliases: hbl
 ---
 
-The 3DS Homebrew Launcher, [boot.3dsx](https://github.com/devkitPro/3ds-hbmenu/releases/download/v2.4.1/boot.3dsx)
+The 3DS Homebrew Launcher, download the latest release zip from here:
```
Comment (lifehackerhansol) [inline cogs/assistance-cmds/boot3dsx.3ds.md:6]: If we're going this route, then we should add a period and start a new sentence for instructions

File: cogs/assistance-cmds/boot3dsx.3ds.md [hunk 2]
```diff
@@ -3,4 +3,8 @@ help-desc: Download link for 3DS Homebrew Launcher, boot.3dsx
...
+
+https://github.com/devkitPro/3ds-hbmenu/releases/
+
+Unzip it to the root of your SD card.
```
Comment (lifehackerhansol) [inline cogs/assistance-cmds/boot3dsx.3ds.md:10]: Add a note to replace what may already exist

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]

**Comment by eip618** [comment]
@lifehackerhansol is there any reason not to link directly to `[project]/releases/latest` in this instance?

**Comment by lifehackerhansol** [comment]
> @lifehackerhansol is there any reason not to link directly to `[project]/releases/latest` in this instance?

No particular reason.

Just know it still needs to be the download page and not the file directly, since dkp seems to add version numbers to their release zips now

**Pull request merged**